exasol_integration_test_docker_environment/test/test_test_environment_nameservers.py

```diff
@@ -51,8 +53,8 @@ def test_single_nameserver(self):
                                                                 [
                                                                     "--nameserver", "'8.8.8.8'",
                                                                 ])
-        self.assert_nameserver("8.8.8.8")
-        close_environments(on_host_docker_environment, google_cloud_docker_environment)
+        with ContextSpawnedTestEnvironment(on_host_docker_environment, google_cloud_docker_environment):
```

Comment:
Usually, you try to generate the object for the context directly in the with statement